src/components/pages/deployments/deploymentDetails/deploymentDetails.js

```diff
@@ -114,18 +118,22 @@ export class DeploymentDetails extends Component {
     } = this.props;
     const {
       id,
-      appliedCount,
-      targetedCount,
-      succeededCount,
-      failedCount,
+      appliedCount = 0,
+      targetedCount = 0,
+      succeededCount = 0,
+      failedCount = 0,
       name,
       priority,
       deviceGroupName,
       createdDateTimeUtc,
-      type,
-      packageName
+      packageType,
+      configType,
+      packageName,
+      customMetrics = {}
     } = currentDeployment;
     const pendingCount = this.state.pendingCount ? this.state.pendingCount : '0';
+    const isADMDeployment = packageType === packageTypeOptions[1];
```

Comment:
This feels fragile. Another package type could be introduced in such an order to affect this.